pkg/docker/client.js

```diff
@@ -407,7 +407,7 @@
                 $(self).off("info", got_info);
```

Comment:
Ack, thanks, will do. I didn't yet go through the packages which ones can drop jquery now. Originally I just wanted to update the documentation, but as a mass-conversion was reasonably fast and easy I thought I'd throw it against the wall and see what sticks. Hence the "WIP".